migrations/20191014173228_migrate_amended_date.js

```diff
@@ -0,0 +1,17 @@
+
+exports.up = function(knex, Promise) {
+  return knex.raw(`UPDATE projects p
+    SET amended_date = (
+        SELECT MAX(pv.updated_at)
+        FROM project_versions pv
+        WHERE pv.project_id = p.id
+        AND pv.status = 'granted'
+        GROUP BY pv.project_id
+        HAVING COUNT(*) > 1
+    )
+  `);
+};
+
+exports.down = function(knex, Promise) {
+  return Promise.resolve();
```

Comment:
It would be useful if the `down` function set all the `amended_date`s to null.

That way if it's incorrect for whatever reason then I can revert and it won't result in incorrect data being displayed.